tests/appsec/test_asm_standalone.py

```diff
@@ -302,11 +384,19 @@ def test_any_upstream_propagation__with_attack__raises_priority_to_2__from_0(sel
                 ["Datadog-Client-Computed-Stats", trueish,] in data["request"]["headers"] for trueish in ["yes", "true"]
             )
 
+        downstream_headers = json.loads(self.r.text)
+        assert downstream_headers["X-Datadog-Origin"] == "rum"
+        assert downstream_headers["X-Datadog-Parent-Id"] != "34343434"
+        assert "_dd.p.other=1" not in downstream_headers["X-Datadog-Tags"]
+        assert "_dd.p.appsec=1" in downstream_headers["X-Datadog-Tags"]
+        assert downstream_headers["X-Datadog-Sampling-Priority"] == "2"
+        assert downstream_headers["X-Datadog-Trace-Id"] == "1212121212121212121"
+
     def setup_any_upstream_propagation__with_attack__raises_priority_to_2__from_1(self):
         trace_id = 1212121212121212121
         parent_id = 34343434
         self.r = weblog.get(
-            "/waf/",
+            "/requestdownstream/",
```

Comment:
based on the method name, it seems like `x-datadog-sampling-priority` should be set to `1` instead of `0`